plugins/resources/compound/libcompound.cpp

```diff
@@ -56,6 +57,37 @@ const std::string AUTO_REPL_POLICY_ENABLED( "on" );
 
 namespace
 {
+    auto throw_error_if_archive_replica_requested(
+            const irods::file_object_ptr& obj,
+            const std::string_view archive_resc_name) -> void
```

Comment:
clang-format?

Reply:
I thought I didn't need to because I was copying from main which had run clang-format but I realize after the fact that I didn't copy some of it but manually edited it.  It is done now.